<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgNewProjectOptionsDialog</class>
 <widget class="QDialog" name="HgNewProjectOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project from Repository</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;New Project from Repository Dialog&lt;/b&gt;
&lt;p&gt;Enter the various repository infos into the entry fields. These values are used, when the new project is retrieved from the repository. If the checkbox is selected, the URL must end in the project name. A repository layout with project/tags, project/branches and project/trunk will be assumed. In this case, you may enter a tag or branch, which must look like tags/tagname or branches/branchname. If the checkbox is not selected, the URL must contain the complete path in the repository.&lt;/p&gt;
&lt;p&gt;For remote repositories the URL must contain the hostname.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;Protocol:</string>
     </property>
     <property name="buddy">
      <cstring>protocolCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="protocolCombo">
     <property name="toolTip">
      <string>Select the protocol to access the repository</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="TextLabel2">
     <property name="text">
      <string>&amp;URL:</string>
     </property>
     <property name="buddy">
      <cstring>vcsUrlEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="vcsUrlEdit">
     <property name="toolTip">
      <string>Enter the url path of the repository (without protocol part)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="vcsUrlButton">
     <property name="toolTip">
      <string>Select the repository url via a directory selection dialog</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="vcsRevisionLabel">
     <property name="text">
      <string>&amp;Revision:</string>
     </property>
     <property name="buddy">
      <cstring>vcsRevisionEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="vcsRevisionEdit">
     <property name="toolTip">
      <string>Enter the revision the new project should be generated from</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextLabel4">
     <property name="text">
      <string>Project &amp;Directory:</string>
     </property>
     <property name="buddy">
      <cstring>vcsProjectDirEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="vcsProjectDirEdit">
     <property name="toolTip">
      <string>Enter the directory of the new project.</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Project Directory&lt;/b&gt;
&lt;p&gt;Enter the directory of the new project. It will be retrieved from 
the repository and be placed in this directory.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="projectDirButton"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="largeCheckBox">
     <property name="text">
      <string>Download all versions of all large files</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="lfNoteLabel">
     <property name="text">
      <string>&lt;b&gt;Note:&lt;/b&gt; This option increases the download time and volume.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>protocolCombo</tabstop>
  <tabstop>vcsUrlEdit</tabstop>
  <tabstop>vcsUrlButton</tabstop>
  <tabstop>vcsRevisionEdit</tabstop>
  <tabstop>vcsProjectDirEdit</tabstop>
  <tabstop>projectDirButton</tabstop>
  <tabstop>largeCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgNewProjectOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgNewProjectOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
